bin/inlined-example.php

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env php
+<?php
+
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+$app = new Application('Example application - inlined version');
+
+$app->add(new class extends Command {
```

Comment:
Yeah, it still breaks the public API though - not a good idea to disable API this way, as it's not reflected in the signature